### PR TITLE
mockup-corpo - task 4 JS

### PR DESCRIPTION
**Pull request by maycatsby**
task 4 https://maycatsby.github.io/ui-internship-lwo2/HTML/mockup-corporate/

### CODE REVIEW

File: HTML/mockup-corporate/js/task4.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import {elementsLoaded} from './elements.js';
+
+export const playPauseButton = () => {
```
Comment (tkachukivan) [inline HTML/mockup-corporate/js/task4.js:3]: maybe something with video would be more descriptive

File: HTML/mockup-corporate/js/task4.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import {elementsLoaded} from './elements.js';
```
Comment (tkachukivan) [inline HTML/mockup-corporate/js/task4.js:1]: elementsLoaded bad name as it is just promise

File: HTML/mockup-corporate/js/task4.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+export const playPauseButton = () => {
+  elementsLoaded.then((data) => {
+    const elements = data;
```
Comment (tkachukivan) [inline HTML/mockup-corporate/js/task4.js:5]: destructuring

File: HTML/mockup-corporate/js/task4.js (hunk 1 above)
Comment (tsvetkovv) [reply HTML/mockup-corporate/js/task4.js:3]: like `initVideoControls`

**Review by tsvetkovv — APPROVED** [review]

**Pull request merged**